app/gui/view/graph-editor/src/lib.rs

```diff
@@ -3022,7 +3022,6 @@ fn init_remaining_graph_editor_frp(
         );
         out.node_added <+ new_node;
         node_to_edit_after_adding <- new_node.filter_map(|&(id,_,do_edit)| do_edit.as_some(id));
-        eval node_to_edit_after_adding((id) model.with_node(*id, |node| node.show_preview()));
```

Comment:
Because the AI Searcher does not want to show a preview. So, this has been moved to the `init_input_node` where we enable this conditionally based on the searcher.